src/components/ebay-carousel/examples/10-autoplay/template.marko

```diff
@@ -0,0 +1,19 @@
+<style>
```

Comment:
Regarding the numbers in the folders -- did you prefer this to sorting?
https://github.com/eBay/ebayui-core/blob/1343c7b2c895669d812694fce693fbc69a7eb842/demo/index.js#L51

Reply:
The ui-marketplace uses folder sorting order which is why I made this update.